pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -17,6 +17,7 @@
 interfaces with C++ for fast linear algebra calculations.
 """
 
+import os.path
 from os import getenv
 from pathlib import Path
```

Comment:
Nitpick: I prefer using `Path` objects for path stuff.

Reply:
Done